js/bootstrap-datepicker.js

```diff
@@ -1273,7 +1279,7 @@
 		_setDate: function(date, which){
 			if (!which || which === 'date')
 				this._toggle_multidate(date && new Date(date));
-			if (!which || which === 'view')
+			if ((!which && this.o.updateViewDate) || which === 'view')
```

Comment:
@vsn4ik `(!which || which === 'view') && this.o.updateViewDate` would make it impossible to change `viewDate` if `updateViewDate` is `false`. Perhaps `_setDate` should be better documented.